legislation-parameters/ipp-parameters.md

```diff
@@ -1,20 +0,0 @@
-# Legislation parameters from IPP
```

Comment:
Are we sure this is **entirely** obsolete ? 
As discussed with @MattiSG, we should not take any decision about bareme ipp within this group of PR, as it deserves its own discussion.